bitacross-worker/enclave-runtime/src/rpc/worker_api_direct.rs

```diff
@@ -74,6 +80,9 @@ pub fn public_api_rpc_handler<Author, GetterExecutor, AccessShieldingKey, OcallA
 	getter_executor: Arc<GetterExecutor>,
 	shielding_key: Arc<AccessShieldingKey>,
 	ocall_api: Arc<OcallApi>,
+	signer: ed25519::Pair,
```

Comment:
I suggested passing `KeyRepository` instead of keys. 
It's possible programaticaly to change the keys during the runtime, see:
`impl<KeyType, SealedIo> MutateKey<KeyType> for KeyRepository<KeyType, SealedIo>`